vt-cli.py

```diff
@@ -7,7 +7,7 @@
 import textwrap
 
 import vt
-from colorama import Fore, init
+from colorama import Fore, init, Back
```

Comment:
I don't see why we have to import `Back` from colorama when its not being used anywhere.

Reply:
I will remove it sorry😁